docs/src/examples/components/Reaction/Types/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
```

Comment:
```suggestion

import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
```

nit: add empty line, please 🐱